pkg/endpoint/endpoint.go

```diff
@@ -272,7 +272,7 @@ type Endpoint struct {
 	realizedRedirects map[string]uint16
 
 	// BPFConfigMap provides access to the endpoint's BPF configuration.
-	bpfConfigMap *bpfconfig.EndpointConfigMap
+	ConfigMap *bpfconfig.EndpointConfigMap
```

Comment:
Renaming to have the first character as a capital letter affects how this field is exposed, including serializing to the filesystem for restore. I don't think we want to do that.

I'm fine with the shorter name, but it should start with a lower case letter (and it's a refactor/cleanup, so not really appropriate for backport to e.g. v1.3 which this bug appears to affect)